packages/lesswrong/server/manualMigrations/index.ts

```diff
@@ -134,3 +134,4 @@ import './2024-02-09-setReviewWinnerCategories'
 import './2024-02-29-markPrivateMessagesAsViewed'
 import './2024-03-01-updateReviewWinnerSocialPreviewImages'
 import './2024-03-06-backfillDefaultVotingSystem'
+import './2024-09-19-importACXMeetups-24-spring'
```

Comment:
This link does indeed seem broken

Reply:
Yep, link is indeed broken. Should be 2024-**03**-19 not 2024-**09**-19. Sorry, I think that's the problem.